posthog/api/insight.py

```diff
@@ -344,6 +356,8 @@ def create(self, validated_data: dict, *args: Any, **kwargs: Any) -> Insight:
         request = self.context["request"]
         tags = validated_data.pop("tags", None)  # tags are created separately as global tag relationships
         team_id = self.context["team_id"]
+        current_url = request.headers.get("Referer")
```

Comment:
Are we sure this `Referer` property is the way to get the current URL? Will proxies / other things affect this?

Reply:
I'm not sure. I don't think proxies will matter much because we don't proxy our API urls. If you have a better suggestion then I'm all ears. I'll also ask in dev.